core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -251,13 +249,12 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants={{.Etcd.SystemdUnitName}}
+        Requires={{.Etcd.SystemdUnitName}}
```

Comment:
Wouldn't modifying this to `Requires` result in cfn-signal.service to be kept failed permanently when `etcd-member.service` failed at first but then restarted via `Restart=on-failure`(according to /usr/lib64/systemd/system/etcd-member.service` and the restart was successful?